GameData/RP-0/Parts/SSTUToolingTanks.cfg

```diff
@@ -305,7 +305,112 @@
 	}
 }
 
-@PART[SSTU-A-Tank-*]:AFTER[RP-0]
++PART[SSTU-SC-TANK-MFT-D]:FOR[RP-0]:NEEDS[SSTU]
```

Comment:
The NEEDS check is not required since patches do not apply if the actual parts are not installed.